src/main/java/org/fenixedu/academic/service/services/scientificCouncil/curricularPlans/EditDegree.java

```diff
@@ -39,8 +39,8 @@
 
     @Atomic
     public static void run(String externalId, String name, String nameEn, String acronym, DegreeType degreeType,
-            Double ectsCredits, GradeScale gradeScale, String prevailingScientificArea, ExecutionYear executionYear)
-            throws FenixServiceException {
+            Double ectsCredits, GradeScale gradeScale, String prevailingScientificArea, ExecutionYear executionYear, String code,
+            String ministryCode) throws FenixServiceException {
```

Comment:
This is an API change, the old signature must be kept (deprecated possibly) along with the new one.